### PR TITLE
Make it possible to add a comment when pushing files

### PR DESCRIPTION
**Pull request by boiko**
This allow for having a richer history when doing changes to packages.

### CODE REVIEW

File: osctiny/extensions/packages.py [hunk 1]
```diff
@@ -237,13 +237,20 @@ def push_file(self, project, package, filename, data):
         :param filename: Name of file
         :param data: content of file
         :type data: str or open file handle
+        :param comment: Optional comment to use as commit message
+
+        .. versionchanged:: {{ NEXT_RELEASE }}
+
+           Added an optional `comment` argument to be used as the commit message when writing the
```
Comment (crazyscientist) [inline osctiny/extensions/packages.py:244]: Sorry, this is RST :wink: 

```suggestion
           Added an optional ``comment`` argument to be used as the commit message when writing the
```

File: osctiny/extensions/packages.py [hunk 2]
```diff
@@ -237,13 +237,20 @@ def push_file(self, project, package, filename, data):
...
+           file.
         """
         path = [self.base_path, project, package, filename]
 
         self.osc.request(
             url=urljoin(self.osc.url, "/".join(path)),
             method="PUT",
-            data=data
+            data=data,
+            params={"comment": comment} if comment else None
```
Comment (crazyscientist) [inline osctiny/extensions/packages.py:253]: I believe it's sufficient to pass the params this way. If `comment` is `None`, `Osc.handle_params` will filter out the item for us :grin: 
```suggestion
            params={"comment": comment}
```

**Review by crazyscientist — CHANGES_REQUESTED** [review]
Thanks for the enhancement. I only have a few minor changes in mind.

**Comment by boiko** [comment]
All resolved.

**Pull request merged**